content/patterns/button/button-pattern.html

```diff
@@ -14,93 +14,94 @@
-              <li>If the button action indicates a context change, such as move to next step in a wizard or add another search criteria, then it is often appropriate to move focus to the starting point for that action.</li>
+              <li>ボタンをアクティブにしても現在のコンテキストが閉じない場合、通常、アクティブ化後もフォーカスはボタンに残ります。例：適用ボタン又は再計算ボタン。</li>
+              <li>ボタンのアクションが、ウィザードの次のステップへの移動や別の検索条件の追加など、コンテキストの変更を示す場合、そのアクションの開始点にフォーカスを移動するのが適切な場合が多いです。</li>
               <li>
-                If the button is activated with a shortcut key, the focus usually remains in the context from which the shortcut key was activated.
-                For example, if <kbd>Alt</kbd> + <kbd>U</kbd> were assigned to an &quot;Up&quot; button that moves the currently focused item in a list one position higher in the list, pressing <kbd>Alt</kbd> + <kbd>U</kbd> when the focus is in the list would not move the focus from the list.
+                ショートカットキーでボタンがアクティブにされた場合、フォーカスは通常、ショートカットキーがアクティブにされたコンテキストに残ります。
+                たとえば、リスト内の現在フォーカスされている項目をリスト内で 1 つ上の位置に移動する「上へ」ボタンに <kbd>Alt</kbd> + <kbd>U</kbd> が割り当てられている場合、リストにフォーカスがあるときに <kbd>Alt</kbd> + <kbd>U</kbd> を押しても、リストからフォーカスは移動しません。
               </li>
             </ul>
           </li>
         </ul>
       </section>
 
       <section id="roles_states_properties">
-        <h2>WAI-ARIA Roles, States, and Properties</h2>
+        <h2>WAI-ARIA ロール、ステート、及びプロパティ</h2>
         <ul>
-          <li>The button has role of <a class="role-reference" href="#button">button</a>.</li>
+          <li>ボタンは <a class="role-reference" href="#button">button</a> のロールを持ちます。</li>
           <li>
-            The <code>button</code> has an accessible label.
-            By default, the accessible name is computed from any text content inside the button element.
-            However, it can also be provided with <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> or <a href="#aria-label" class="property-reference">aria-label</a>.
+            <code>button</code> はアクセシブルなラベルを持ちます。
+            デフォルトでは、アクセシブルネームはボタン要素内のテキストコンテンツから計算されます。
+            ただし、<a href="#aria-labelledby" class="property-reference">aria-labelledby</a> 又は <a href="#aria-label" class="property-reference">aria-label</a> を使用して提供することもできます。
           </li>
-          <li>If a description of the button's function is present, the button element has <a href="#aria-describedby" class="property-reference">aria-describedby</a> set to the ID of the element containing the description.</li>
-          <li>When the action associated with a button is unavailable, the button has <a class="state-reference" href="#aria-disabled">aria-disabled</a> set to <code>true</code>.</li>
+          <li>ボタンの機能の説明が存在する場合、ボタン要素は説明を含む要素の ID に <a href="#aria-describedby" class="property-reference">aria-describedby</a> が設定されます。</li>
+          <li>ボタンに関連付けられたアクションが利用できない場合、ボタンは <a class="state-reference" href="#aria-disabled">aria-disabled</a> が <code>true</code> に設定されます。</li>
```

Comment:
「設定されます」は、勝手にやってくれるように読めてしまう。それと、属性を誰が持つのか明確にするために、「ボタン要素の aria-describedby」という書き方にするのがよさそう。

```suggestion
          <li>ボタンの機能の説明が存在する場合、説明を含む要素の ID を ボタン要素の <a href="#aria-describedby" class="property-reference">aria-describedby</a> に設定します。</li>
          <li>ボタンに関連付けられたアクションが利用できない場合、ボタンの <a class="state-reference" href="#aria-disabled">aria-disabled</a> を <code>true</code> に設定します。</li>
```